languages/go/tests/parity_test.go

```diff
@@ -145,13 +145,8 @@ func (MethodVariants) GetEmptyIter() interfaces.Iterator {
 	return IterableClass{}
 }
 
-// TODO: I don't think these make sense. Maybe as interfaces?
 type ParentClass struct{}
```

Comment:
Is `ParentClass` still buying us anything in the absence of `ChildClass` & `GrandchildClass`?

Reply:
it's used as a stand-in `other` type to be compared against `UnitType` in a comparison parity unit-test.